### DISCUSSION TITLE
Adjusting styling of collapse

### DISCUSSION THREAD

**Discussion by cjc-dev** [saadeghi/daisyui#1110] [General]
Hey, I'm trying to reduce the overall height/chunkiness of the `collapse-title` element but had no luck so far. What's the best approach for eliminating or reducing the amount of padding on the `collapse-title`? I've made an attempt [here on this tailwind code pen](https://play.tailwindcss.com/5RRzTKmi7T) but it wasn't successful.

**Comment by saadeghi** [comment]
I applied some changes in the latest version (`2.26.0`)  
After update, you can add `p-0 min-h-0` to both checkbox and `collapse-title` to remove `padding` and `min-height`  

https://play.tailwindcss.com/yALPVZ1961
```html
<div tabindex="{0}" class="collapse w-full collapse-arrow">
  <input type="checkbox" class="p-0 min-h-0"/>
  <div class="collapse-title p-0 min-h-0 text-black font-bold uppercase bg-red-200">TITLE</div>
  <div class="collapse-content">
    <ul>
      <li class="link link-hover">'EXAMPLE HIDDEN CONTENT'</li>
    </ul>
  </div>
</div>
```

**Reply by abcbrookie** [reply]
Blessss thank you so much !
